### PR TITLE
Update docs, docs CI, and example readmes

### PR DESCRIPTION
**Pull request by haipinglu**
### Description
- Update (outdated) readmes for two examples `cifar_isonet` and `digits_dann_lightn`
- Update docs requirements to fix missing docs
- Add pytorch tests to tests readme
- Use RTD docs CI instead
### Status
**Ready**

### Types of changes

- [x] Non-breaking change (fix or new feature that would not break existing functionality).

### CODE REVIEW

File: docs/source/kale.embed.rst [hunk 1]
```diff
@@ -30,7 +29,6 @@ kale.embed.gcn module
    :members:
    :undoc-members:
    :show-inheritance:
-   :exclude-members: message, update
```
Comment (pz-white) [inline docs/source/kale.embed.rst:33]: May I ask why the exclude-members are deleted? In my understanding the message and update should be private methods and not be emerged in docs. I follow the rules on pytorch-geometrics doc: https://pytorch-geometric.readthedocs.io/en/latest/modules/nn.html#torch_geometric.nn.conv.GCNConv

File: docs/source/kale.prepdata.rst [hunk 2]
```diff
@@ -31,7 +31,7 @@ kale.prepdata.tensor\_reshape module
    :undoc-members:
    :show-inheritance:
 
-kale.prepdata.chem\_transform module
+kale.prepdata.chem_transform module
```
Comment (pz-white) [inline docs/source/kale.prepdata.rst:34]: Does this change work on doc build? As I remeber it failed without "\" on my local build.

File: docs/source/kale.utils.rst [hunk 3]
```diff
@@ -6,7 +6,7 @@ Utilities
 Submodules
 ----------
 
-kale.utils.csv\_logger module
+kale.utils.csv_logger module
```
Comment (pz-white) [inline docs/source/kale.utils.rst:9]: Same as above.

File: docs/source/kale.embed.rst (hunk 1 above)
Comment (haipinglu) [reply docs/source/kale.embed.rst:33]: I have thought it was the cause of the missing docs. I proved that I was wrong. It was due to the libraries. I will reverse and test.

File: docs/source/kale.prepdata.rst (hunk 2 above)
Comment (haipinglu) [reply docs/source/kale.prepdata.rst]: It seems work both ways on rtd. I made the changes trying to find out why docs are missing on rtd

**Review by pz-white — APPROVED** [review]
All tests are passed! This PR LGTM, thanks Haiping!

**Pull request merged**